pom.xml

```diff
@@ -0,0 +1,189 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<modelVersion>4.0.0</modelVersion>
+
+<groupId>io.github.dftrakesh</groupId>
+<artifactId>lightspeed</artifactId>
+<version>1.0</version>
+<packaging>jar</packaging>
+
+<name>lightspeed</name>
+<description>LightSpeed API using JDK 11</description>
+<url>https://github.com/dftrakesh/shipstation</url>
```

Comment:
Change url to lightspeed instead of shipstation

Reply:
ok sir